string tid
string classification
float64[] do_coordinate
float64[] do_heading
float64[] do_dimension
